coralproject/talk
# Index Creation Patch

## What does this PR do?

Patches bug related to index creation causing a circular import error (See https://www.pivotaltracker.com/n/projects/1863625/stories/157098971)

## How do I test this PR?

1. Create a `plugins.json` containing either `talk-plugin-facebook-auth` or `talk-plugin-google-auth` and start the server.
2. Notice it doesn't crash!
